detox/src/ipc/SessionState.test.js

```diff
@@ -0,0 +1,69 @@
+const SessionState = require('./SessionState');
+
+describe('SessionState', () => {
+  /** @type {SessionState} */
+  let sessionState;
+
+  beforeEach(() => {
+    sessionState = new SessionState({
```

Comment:
Just to make sure, is there a function reviving test?

Reply:
Nope, I'll add it

Reply:
Thinking about this, do we have any use case where one of the stringified object props is a function?

Reply:
Yes, unfortunately. Many Bunyan formatting options are passed as functions, e.g. `showDate: (date) => ...` and all prefixers/stringifiers too.

Reply:
done